src/test/java/org/kizombadev/markdownparser/SyntaxAnalyzerTest.java

```diff
@@ -165,4 +166,33 @@ public void test() {
         assertThat(paragraph.getChildren().get(0).getChildren().get(0)).isTextElementWith("Foo");
         assertThat(paragraph.getChildren().get(1)).isTextElementWith(" Bar");
     }
+
+    @Test
+    @Ignore
+    public void testBug() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)